Client/src/services/saveLastCityToLS.js

```diff
@@ -0,0 +1,26 @@
+const saveLastCityToLS = (city, id) => {
+  let stuffInLs = JSON.parse(localStorage.getItem('lastCity'));
```

Comment:
1) Please avoid using non-official lingo like `stuff`. lastCity whoulb be enough
2) why let? will it change? will you reassign later? 
3) why are you iterating through something that you got by 'lastCity' key. not lastCities ?
4) JSON.parse may fail - so use try to always use try catch block for it